.github/workflows/build-images-beta.yaml

```diff
@@ -18,6 +18,15 @@ permissions:
   id-token: write
 
 jobs:
+  echo-inputs:
+    if: ${{ github.event_name == "workflow_dispatch" }}
+    name: Echo Workflow Dispatch Inputs
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Echo Workflow Dispatch Inputs
+        run: |
+          echo '${{ tojson(inputs) }}'
+
```

Comment:
Why does this change `build-images-beta.yaml` but not the other build images workflows?

Reply:
Ah, because that's the only job with workflow dispatch:

```
$ git grep -l workflow_dispatch .github/workflows/build*
.github/workflows/build-images-beta.yaml
`

Reply:
Yep exactly :+1: